labelbox/schema/task.py

```diff
@@ -65,3 +67,12 @@ def wait_till_done(self, timeout_seconds=300) -> None:
             timeout_seconds -= check_frequency
             time.sleep(sleep_time_seconds)
             self.refresh()
+
+    def get_result(self) -> str:
+        """ Downloads the result file from Task
+        """
+        if self.result:
+            response = requests.get(self.result)
+            response.raise_for_status()
+            return response.text
+        return ""
```

Comment:
it might be good to return None vs an empty string. That way an empty response is clearly different from a result that doesn't exist.